content/documentation/tutorials/nats-queueing.md

```diff
@@ -44,7 +44,7 @@ node node-sub.js foo my-queue
 **4. Run the Ruby client subscriber with queue group name.**
 
 ```
-nats-sub foo my-queue &
+ruby queue_sub.rb foo my-queue &
```

Comment:
It should actually be this here instead:
```
gem install nats
nats-queue foo my-queue &
```

Reply:
ruby queue_sub.rb foo my-queue & command gives the below output:

[root@NATS examples]# ruby queue_sub.rb foo my-queue &
[2] 23243
[root@NATS examples]# Listening on [foo], queue group [my-queue]

Reply:
while in nats-sub foo my-queue & command the client only listens on a subject foo without entering any queue group as follows:

[root@NATS examples]# nats-sub foo my-queue &
[3] 23281
[root@NATS examples]# Listening on [foo]

Reply:
uby queue_sub.rb foo my-queue & command gives the below output:

[root@NATS examples]# ruby queue_sub.rb foo my-queue &
[2] 23243
[root@NATS examples]# Listening on [foo], queue group [my-queue]

Reply:
while in nats-sub foo my-queue & command the client only listens on a subject foo without entering any queue group as follows:

[root@NATS examples]# nats-sub foo my-queue &
[3] 23281
[root@NATS examples]# Listening on [foo]